src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -77,6 +78,9 @@ public void execute(Parameters parameters) {
 
     @Override
     public void apply(Project project) {
+        if (project.getRootDir().toString().toUpperCase().contains("ONEDRIVE")) {
```

Comment:
I'm still a bit worried about if this causes an issue when working in other languages. Wrap the contains portion in its own try catch that just catches and ignores any exception. But make sure the thrown OneDriveException isn't contained in that try catch.

Reply:
only place I could see it getting ignored is in checkBuildFailed. We might want to add a clause when checking for exceptions about this

Reply:
Apply is called at the very beginning. This is the right place to put it.

Reply:
If it’s not clear, I’m worried about toUpperCase throwing a weird exception in other languages. So if we try catch any exception potentially thrown by that whole statement, we cover our bases.

Reply:
I cant find anywhere else where the error would be ignored so should be good

Reply:
I’m saying I want an error to be ignored if one is thrown.

Reply:
Working with other languages is difficult. I don’t want any of the string comparisons throwing a runtime exception and causing the build to not work. We want this change to be the least invasive as possible.

Reply:
Ohh ok say it should be wrapped in a try catch?

Reply:
Just the whole statement doing the comparison. You would need to throw the onedrive exception outside of that try catch so it’s not accidentally caught.

Reply:
No. That is not what I want.

Reply:
Right ok i'll fix that in my last commit

Reply:
Ok should be fixed now